daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md

```diff
@@ -51,14 +51,20 @@ spec:
 |--------------------|:--------:|------------|-----|---------|
 | topics | N | Input | A comma-separated string of topics. | `"mytopic1,topic2"` |
 | brokers | Y | Input/Output | A comma-separated string of Kafka brokers. | `"localhost:9092,dapr-kafka.myapp.svc.cluster.local:9093"` |
+| clientID            | N | Input/Output | A user-provided string sent with every request to the Kafka brokers for logging, debugging, and auditing purposes. | `"my-dapr-app"` |
 | consumerGroup | N | Input | A kafka consumer group to listen on. Each record published to a topic is delivered to one consumer within each consumer group subscribed to the topic. | `"group1"` |
+| consumeRetryEnabled | N | Input/Output | Enable consume retry by setting to `"true"` | `"true"`, `"false"` |
```

Comment:
Because the default value of this metadata is different in binding and pubsub, suggest to add:

Default to false in kafka binding component.